src/System.Windows.Forms/tests/UnitTests/System/Windows/Forms/ToolStripContainer.ToolStripContainerAccessibleObjectTests.cs

```diff
@@ -0,0 +1,68 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+using static Interop.UiaCore;
+
+namespace System.Windows.Forms.Tests
+{
+    public class ToolStripContainer_ToolStripContainerAccessibleObjectTests : IClassFixture<ThreadExceptionFixture>
+    {
+        [WinFormsFact]
+        public void ToolStripContainerAccessibleObject_Ctor_Default()
+        {
+            using ToolStripContainer toolStripContainer = new();
+            ToolStripContainer.ToolStripContainerAccessibleObject accessibleObject = new(toolStripContainer);
+
+            Assert.Equal(toolStripContainer, accessibleObject.Owner);
+            Assert.False(toolStripContainer.IsHandleCreated);
+        }
+
+        [WinFormsTheory]
+        [InlineData((int)UIA.NamePropertyId, "TestName")]
+        [InlineData((int)UIA.AutomationIdPropertyId, "ToolStripContainer1")]
+        public void ToolStripContainerAccessibleObject_GetPropertyValue_Invoke_ReturnsExpected(int propertyID, object expected)
+        {
+            using var control = new ToolStripContainer
+            {
+                Name = "ToolStripContainer1",
+                AccessibleName = "TestName"
+            };
+
+            Assert.False(control.IsHandleCreated);
+            var accessibleObject = new ToolStripContainer.ToolStripContainerAccessibleObject(control);
+            object value = accessibleObject.GetPropertyValue((UIA)propertyID);
+
+            Assert.Equal(expected, value);
+            Assert.False(control.IsHandleCreated);
+        }
+
+        [WinFormsFact]
+        public void ToolStripContainerAccessibleObject_GetPropertyValue_Focused_ReturnsExpected()
+        {
+            using var control = new ToolStripContainer();
+
+            Assert.False(control.IsHandleCreated);
+            var accessibleObject = new ToolStripContainer.ToolStripContainerAccessibleObject(control);
```

Comment:
```suggestion
            var accessibleObject = new ToolStripContainer.ToolStripContainerAccessibleObject(control);

            Assert.False(control.IsHandleCreated);

```
I think the check for the `Handle` is more correct to do after initiating the `AccessibleObject`